starcat/starcatalog.py

```
@@ -303,6 +303,13 @@ def __str__(self) -> str:
 
         return ret
 
+    # This is a stupid thing to do, but it's necessary to avoid mypy from complaining
+    # about missing attributes. mypy ignores attributes for classes that have a
+    # __getattr__ method.
+    @no_type_check
+    def __getattr__(self, name: str) -> Any:
```

Ruff (0.14.4)

Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)